tests/plugins/plugin.resume.test.js

```diff
@@ -0,0 +1,51 @@
+/* global BookReader */
+import '../../BookReader/jquery-1.10.1.js';
+import '../../BookReader/jquery-1.10.1.js';
+import '../../BookReader/jquery-ui-1.12.0.min.js';
+import '../../BookReader/jquery.browser.min.js';
+import '../../BookReader/dragscrollable-br.js';
+import '../../BookReader/jquery.colorbox-min.js';
+import '../../BookReader/jquery.bt.min.js';
+
+import '../../BookReader/BookReader.js';
+import '../../src/js/plugins/plugin.resume.js';
+
+let br;
+beforeAll(() => {
+  document.body.innerHTML = '<div id="BookReader">';
+  br = new BookReader();
+  br.init();  
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('Plugin: Remember Current Page in Cookies', () => {
```

Comment:
Then these `docCookies` tests can all go in their own file :)

Reply:
Done.